modules/021-cni-cilium/images/cilium/patches/README.md

```diff
@@ -5,21 +5,3 @@
 Helps with handling LoadBalancer/NodePort traffic to hostNetwork endpoints.
```

Comment:
Remove the patch and corresponding Readme line?

Reply:
I don't get it.(